controllers/errors.js

```diff
@@ -1,8 +1,25 @@
-exports.handle404s = (req, res, next) => {
-  res.status(404).send({ msg: "Cannot find what you wanted" });
+exports.handleCustomErrors = (req, res, next) => {
+  res
+    .status(404)
+    .send({ msg: "You tried to look for something and it wasn't found" });
+};
+
+exports.handle400s = (err, req, res, next) => {
+  if (err.status === 400) {
+    res.status(400).send({ msg: "Bad request" });
```

Comment:
In the case of a 400 error - this should usually be something that is thrown by psql  ( putting a string where it expects a number will break the query ) - in which case you do not need to throw your own errors for this as psql will throw it for you - look for different `err.code` values which come back on psql errors